tests/test_e2e_01_kytos_startup.py

```diff
@@ -122,4 +122,15 @@ def test_start_kytos_api_core(self):
     @pytest.mark.skipif(not os.path.exists('/var/log/syslog'), reason="/var/log/syslog does not exist")
     def test_start_kytos_without_errors(self):
         with open(self.logfile, "r") as f:
-            assert re.findall(r'kytos.*?(error|exception)(.*)?', f.read(), re.I) == []
+            entries = re.findall(r'kytos.*?(error|exception)(.*)?', f.read(), re.I)
+        known_logs = [
```

Comment:
I got the idea of `known_logs` as logs to be ignored, but I didn't get why we had to start ignoring these logs from uvicorn or was the intention to update the findall regex expr to start matching other errors|excs in any other (non kytos) modules too?